libmcount/mcount.c

```diff
@@ -1272,6 +1280,7 @@ static void __attribute__((constructor))
 mcount_init(void)
 {
 	mcount_startup();
+	python_init();
```

Comment:
What do you think about calling `python_init()` in `mcount_startup()` ?
using a if statement `if (pyfile) {` like other options

e.g.
```
1066         if (patch_str)
1067                 mcount_dynamic_update(&symtabs, patch_str);
```